### PR TITLE
Button like

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**

### CODE REVIEW

**Review by cartige — DISMISSED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "axios": "^1.2.1",
     "prop-types": "^15.8.1",
+    "props": "^0.3.0",
```
Comment (LoicBrassart) [inline frontend/package.json]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
14 files for a Like button ? Should be good! :sweat_smile: 
(Don't update files unrelated to your current US, it can only bring a lot of new bugs)

Plz fix Evil :(

File: frontend/package.json (hunk 1 above)
Comment (PriscilliaAmmeux) [reply frontend/package.json]: sorry, i will uninstall

Comment (PriscilliaAmmeux) [reply frontend/package.json]: ok, i am notting. :s i know, I messed up for the 14 files :s

**Review by cartige — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Fostrankill — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SNE

**Pull request merged**